app/response_processor.py

```diff
@@ -70,7 +70,11 @@ def process(self, encrypted_survey):
             decrypted_json['invalid'] = True
 
         self.store_survey(decrypted_json)
-        self.send_receipt(decrypted_json)
+        if decrypted_json["survey_id"] != "feedback":
```

Comment:
You could use decrypted_json.get("survey_id") to avoid key errors if a future survey comes in without one